skopt/tree_opt.py

```diff
@@ -112,7 +112,7 @@ def _tree_minimize(func, dimensions, base_estimator, n_calls,
 
 
 def gbrt_minimize(func, dimensions, base_estimator=None, n_calls=100,
-                  n_points=20, n_random_starts=10, x0=None, y0=None,
+                  n_points=1000, n_random_starts=10, x0=None, y0=None,
```

Comment:
Since `n_points` is random anyway, why not set it to 10000 for consistency with Spearmint and SMAC?